roadmaps/2022.md

```diff
@@ -91,33 +91,73 @@ The core team has two main goals for this quarter: to improve the product and al
 
 ### Team
 
-- [ ] Finish the recruitment process to hire a senior software engineer.
-- [ ] Onboard the new software engineer.
-- [ ] Adapt the team synchronization and rhythm according to the Product Squad (PS) freshly introduced by the product team.
+- [X] Finish the recruitment process to hire a senior software engineer.
+- [ ] Onboard the new software engineer -> The person arrives next quarter.
+- [X] Adapt the team synchronization and rhythm according to the Product Squad (PS) freshly introduced by the product team -> We had some delay but we found out the new way of working with Product Squad better fits the needs of the team.
 
 ### Misc
 
-- [ ] Finish internship work on Jayson. Integrate it to one or two routes on the Meilisearch side.
-- [ ] Work with the devrel team to promote the contributions we need on the tokenizer side.
-- [ ] Keep improving indexation speed/crash
-  - [ ] by gathering user feedback and use-cases and creating benchmarks according to them: we'll ensure we succeeded or not the improvement.
-  - [ ] by keep investigating and trying new technical solutions on our side.
+- [ ] Finish internship work on Jayson. Integrate it to one or two routes on the Meilisearch side. -> We don't know yet if we will prioritize it in Q4 2022 due to the huge breaking it can lead.
+- [X] Work with the devrel team to promote the contributions we need on the tokenizer side -> done during Hacktoberfest, so Q4 2022.
+- [X] Keep improving indexation speed/crash
+  - [ ] by gathering user feedback and use-cases and creating benchmarks according to them: we'll ensure we succeeded or not the improvement -> We are working on creating use-case benchmarks in Q4 2022.
+  - [X] by keep investigating and trying new technical solutions on our side -> we made a lot of changes described in [this issue](https://github.com/meilisearch/meilisearch/issues/2132).
+
+## Q4 2022
+
+This quarter is the last one we have to prepare v1 in good condition. This way, we have prioritized tasks that would help the team deliver Meilisearch v1: a stable and impactful version of Meilisearch.
+It also means we will de-prioritize implementations that could impact in a bad way the critical features of Meilisearch, like improving the indexing speed in a way that would risk making the indexing fail.
+
+Q4 is also the quarter where the Hacktoberfest is happening and this will lead us to spend time on it. We created a lot if issues during Q3, especially regarding the language support, and we hope attract the community and make them contribute in October.
+
+### v1 work
+
+- [ ] Be production ready for v1
+  - [ ] Release the missing production-ready features: index swap, cancel/delete of tasks, and other missing features noticed by the product team.
+  - [ ] Ensure we don't have any bugs/crashes impacting the stability of Meilisearch. The RAM consumption should be appropriate when indexing and searching, or at least we must be transparent with the users about it.
```

Comment:
I'd just like to say that I am quite worried about this one. I think it is both very important and it is going to take a significant amount of work to get right.

Reply:
The question is what it means "to get right". The crucial point we need to solve here first is transparency with the users. I will try to rephrase to make the right order of priority